pulp_smash/tests/pulp3/utils.py

```diff
@@ -218,3 +218,14 @@ def read_repo_content(repo):
     return (api
             .Client(config.get_config(), api.json_handler)
             .get(urljoin(get_latest_repo_version(repo), 'content/')))
+
+
+def get_name_content_units(repo):
+    """Return the name of content units present in a given repository.
+
+    :param repo: A dict of information about the repository.
+    :returns: A list with the names of units present in a given repository.
+    """
+    return [k
+            for content_unit in read_repo_content(repo)['results']
+            for k, v in content_unit['artifacts'].items()]
```

Comment:
Since each value is discarded, use `keys()` instead of `items()`.

Reply:
Also, can some other way of writing this logic be devised? This hurts my brain. A pair of generator functions would be more efficient and easier to understand.

Reply:
Also also, please avoid hanging indents like these. It's a stylistic convention that's avoided almost everywhere else in the Pulp Smash code base.